addons/payment/models/payment_acquirer.py

```diff
@@ -9,8 +9,9 @@
 import psycopg2
 
 from odoo import api, exceptions, fields, models, _, SUPERUSER_ID
+from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
```

Comment:
Please create a second [separate block for Odoo module imports](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports).